frontend/src/registerServiceWorker.js

```diff
@@ -10,20 +10,17 @@ if (process.env.NODE_ENV === 'production') {
     updated() {
       store.commit(SET_UPDATE_READY, true);
     },
-    // ready() {
-    //   console.log('App is being served from cache by a service worker.\n' +
-    //     'For more details, visit https://goo.gl/AFskqB');
-    // },
-    // cached() {
-    //   console.log('this');
-    //   console.log('Content has been cached for offline use.');
-    // },
-    // offline() {
-    //   console.log('No internet connection found. App is running in offline mode.');
-    // },
-
-    // error(error) {
-    //   console.error('Error during service worker registration:', error);
-    // },
+    ready() {
```

Comment:
probably just remove?

Reply:
I forced the SW to automatically refresh so we can get rid of the Refresh component.